lib/commons/dom/get-rect-stack.js

```diff
@@ -124,13 +126,27 @@ export function getRectStack(grid, rect, recursed = false) {
         const rectX = clientRect.left;
         const rectY = clientRect.top;
 
+        if (gridCellNode.attr('id') === 'debugNode') {
+          console.log(JSON.stringify({
+            el: gridCellNode.actualNode.cloneNode(false).outerHTML,
+            y: y,
+            floorY: floorY,
+            'rectY': rectY,
+            height: clientRect.height,
+            'rectY+height': rectY + clientRect.height,
+            'floor(rectY+height)': floor(rectY + clientRect.height)
+          },null,2));
+        }
+
```

Comment:
```suggestion
```

Reply:
Ops. forgot I still had those in when trying to figure out why IE11 was broken.